File: csharp/Ice/plugin/LoggerI.cs
// Copyright (c) ZeroC, Inc.

using System;

internal class LoggerI : Ice.Logger
{
    public void print(string message)
    {
        Console.WriteLine("PRINT: " + message);
    }

    public void trace(string category, string message)
    {
        Console.WriteLine("TRACE(" + category + "): " + message);
    }

    public void warning(string message)
    {
        Console.WriteLine("WARNING: " + message);
    }

    public void error(string message)
    {
        Console.WriteLine("ERROR: " + message);
    }

    public string getPrefix()
    {
        return "";
    }

    public Ice.Logger cloneWithPrefix(string prefix)
    {
        return new LoggerI();
    }
}

File: java/Ice/plugin/LoggerI.java
// **********************************************************************
//
// Copyright (c) 2003-2016 ZeroC, Inc. All rights reserved.
//
// **********************************************************************

public class LoggerI implements com.zeroc.Ice.Logger
{
    @Override
    public void print(String message)
    {
        System.out.println("PRINT: " + message);
    }
    
    @Override
    public void trace(String category, String message)
    {
        System.out.println("TRACE(" + category + "): " + message);
    }
    
    @Override
    public void warning(String message)
    {
        System.out.println("WARNING: " + message);
    }

    @Override
    public void error(String message)
    {
        System.out.println("ERROR: " + message);
    }

    @Override
    public String getPrefix()
    {
        return "";
    }

    @Override
    public com.zeroc.Ice.Logger cloneWithPrefix(String prefix)
    {
        return new LoggerI();
    }
}
